Documentation/platform/strategic-merge-patch.md

```diff
@@ -118,3 +118,214 @@ spec:
   - name: "scrape.timestamp-tolerance"
     value: "15ms"
 ```
+
+## How To Deploy Prometheus In Restricted Or Baseline Namespace
```

Comment:
The secion should also mention that if the Thanos sidecar is enabled, the same changes should be applied for the `thanos-sidecar` container?